features/step_definitions/toast_steps.rb

```diff
@@ -5,12 +5,14 @@
 end
 
 Then /^(?:they )?should see a(?: )?(\w+)? toast alert with text "(.*)"$/ do |type, message|
+  find '.pf-c-alert-group.pf-m-toast'
```

Comment:
You need both, `find` + `within`? Is it now `find` already performed by `within`?

Reply:
So, `find` does wait for the element to appear, but `within` doesn't.

So, my guess is that the flow is the following:
1. Some button is pressed
2. The page gets reloaded
3. Once the page is fully loaded, a toast message appears

So, my guess is that `within` is sometimes performed when the page is not fully loaded yet (still blank maybe?), so it fails as soon as it can't find the selector, without waiting. This is why I am trying to add an extra `find`.

Reply:
Makes sense :+1: